Gemfile.lock

```diff
@@ -114,7 +114,7 @@ DEPENDENCIES
   pry
   pry-byebug
   pry-stack_explorer
-  qpid_proton
+  qpid_proton (~> 0.17.0)
```

Comment:
This causes a failure using the vagrant image due to what appears to be a conflict with the rpm installed qpid_proton 0.18 on Centos/7. If I updated both the gemfile & temfile.lock to (~> 0.18) it worked for me with vagrant up.

Reply:
Okay, I'll just scrap this change then since I'm not precisely sure the underlying reason for it.

Reply:
I thought we had to stick with 0.17 for some reason?  Fedora is up to 0.18 right now, so I'd be happy to move to it as long as it doesn't break anything.

To be honest, the qpid spec tests fail pretty frequently when I run them locally anyway.  Maybe we could move them to a specific test suite for Jenkins only or make them optional?  This qpid gem has been a chronic pain in the neck for awhile now.